.github/workflows/release.yaml

```diff
@@ -17,14 +17,14 @@ jobs:
 
   build:
     name: Build charm
-    uses: canonical/data-platform-workflows/.github/workflows/build_charm.yaml@v16.7.0
+    uses: canonical/data-platform-workflows/.github/workflows/build_charm.yaml@v17.0.1
 
   release:
     name: Release charm
     needs:
       - ci-tests
       - build
-    uses: canonical/data-platform-workflows/.github/workflows/release_charm.yaml@v16.7.0
+    uses: canonical/data-platform-workflows/.github/workflows/release_charm.yaml@v17.0.1
```

Comment:
Can we move to `v21.0.1` instead?

We are planning to do it for opensearch as well, here: https://github.com/canonical/opensearch-operator/pull/432

Reply:
We can move to any, but pls note that once Opensearch is on 2.16 OSD pipelines will break if this PR is not merged... While I'm on hoidays ;-) 

The above is surely fixing the broken Release pipelines we've got, which was the main objective.

I change it now, but I may ask you to make sure hte PR lands before releasing Opensearch 2.16.